src/Quaternion.jl

```diff
@@ -118,7 +118,7 @@ function exp(q::Quaternion)
     if th > 0
         scale *= sin(th) / th
     end
-    Quaternion(se * cos(th), scale * q.v1, scale * q.v2, scale * q.v3, abs(s) < eps(typeof(s)))
+    Quaternion(se * cos(th), scale * q.v1, scale * q.v2, scale * q.v3, abs(s) < eps(typeof(se)))
```

Comment:
Yes, I think `iszero` is more reasonable. Nowhere else in the code base is approximate equality used to establish whether a quaternion is normalized, and `norm` indicates structural normalization, which I think fits better with exact equality.

Reply:
It's a little grey whether this would be a breaking change, but because this line of code is currently inconsistent with how `norm` is interpreted elsewhere in the codebase, I think this is a fine change to make.

Reply:
Thanks for the review! I've updated with `iszero`.